qiskit/transpiler/passes/scheduling/time_unit_conversion.py

```diff
@@ -25,15 +25,15 @@ class TimeUnitConversion(TransformationPass):
     """Choose a time unit to be used in the following time-aware passes,
     and make all circuit time units consistent with that.
 
-    This pass will add a .duration metadata to each op whose duration is known,
+    This pass will add a ``.duration`` metadata to each op whose duration is known
```

Comment:
I think the most precise link here is `` :attr:`.Instruction.duration` ``, so

> This pass will add an `` :attr:`.Instruction.duration` `` attribute to each op ...

Reply:
good one. done in 0daa2693084e71e0d4c7c0b560153c3447dfe835